nginx/config.go

```diff
@@ -293,6 +293,17 @@ http {
 		proxy_pass {{$builderConfig.ServiceIP}}:2222;
 	}
 }{{ end }}
+
+{{ if $routerConfig.StatusEndpoint }}
+  sevrer {
```

Comment:
Unless @mboersma objects, I am unopposed to `router.<domain>` (or maybe `deis-router.<domain>`) becoming the router's official address, much as `deis.<domain` is the address of the controller and `deis-builder.<domain>` is the address of the builder. We'd have to add `router` (or `deis-router`) to the list of reserved names in the controller.

That being said... that can be implemented independently (i.e. in another PR) of this feature. `router.<domain>` will _already_ get you to the efault vhost today _unless_ you have a userland app called `router`. Giving the router its own domain is just a formality of sorts. So, while I am unopposed, let's consider it on its own merits. Would you mind opening an issue?

In the meantime, this PR can proceed just by moving the `/nginx_status` endpoint to the existing default vhost.